ConsoleView.h

```diff
@@ -17,32 +18,31 @@ class ConsoleView
 {
 
 public:
-    ConsoleView(){};
```

Comment:
I'm not sure if C++ requires a constructor? I'm not totally sure, but if it compiles without a constructor thats fine too (it would just be a default constructor that does nothing anyways)

Reply:
Think I should just delete it?

Reply:
You can just keep it as ConsoleView(); in the header file and then in the cpp file you can just implement it as a blank constructor
`ConsoleView::ConsoleView()
{}`

Reply:
After adding ConsoleView::ConsoleView() {} to the .cpp file i got an error
 implicitly-declared ‘constexpr ConsoleView::ConsoleView()’